test/test_transforms.py

```diff
@@ -224,31 +222,31 @@ def test_librosa_consistency(self):
                                                             hop_length=hop_length,
                                                             power=2)
 
-        out_torch = spect_transform(sound).squeeze().cpu().numpy().T
-        self.assertTrue(np.allclose(out_torch, out_librosa, atol=1e-5))
+        out_torch = spect_transform(sound).squeeze().cpu().t()
+        self.assertTrue(torch.allclose(out_torch, torch.from_numpy(out_librosa), atol=1e-5))
 
         # test mel spectrogram
         melspect_transform = torchaudio.transforms.MelSpectrogram(sr=sample_rate, window=torch.hann_window,
                                                                   hop=hop_length, n_mels=n_mels, n_fft=n_fft)
         librosa_mel = librosa.feature.melspectrogram(y=sound_librosa, sr=sample_rate,
                                                      n_fft=n_fft, hop_length=hop_length, n_mels=n_mels,
                                                      htk=True, norm=None)
-        torch_mel = melspect_transform(sound).squeeze().cpu().numpy().T
+        torch_mel = melspect_transform(sound).squeeze().cpu().t()
 
         # lower tolerance, think it's double vs. float
-        self.assertTrue(np.allclose(torch_mel, librosa_mel, atol=5e-3))
+        self.assertTrue(torch.allclose(torch_mel.type(torch.DoubleTensor), torch.from_numpy(librosa_mel), atol=5e-3))
```

Comment:
Use torch.double instead of torch.DoubleTensor.

Since you're using torch.from_numpy and melspect_transform is returning a numpy array, you still need numpy to execute this code, but this comes via the librosa dependency.

Flag this test as skipped via [@unittest.skip](https://docs.python.org/2/library/unittest.html#skipping-tests-and-expected-failures) if librosa and scipy can't be imported and instead of returning as it currently does. Also add some docs that mention this.

Again check if we can use equal instead of all close.